anna-sala/challenges/toh-full-stack-ddbb/frontend/src/redux/actions/actionCreators.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable no-underscore-dangle */
+import axios from 'axios';
+import actionTypes from './actionTypes';
+
+const url = 'http://localhost:2021/heroes';
+
+export function loadHeroes() {
+  return async (dispatch) => {
+    try {
+      const { data } = await axios(url);
+      dispatch({
+        type: actionTypes.LOAD_HEROES,
+        heroes: data,
+      });
+    } catch (error) {
+      dispatch({
+        type: 'LOAD_HEROES_ERROR',
```

Comment:
(Esto huele a junior) Please use a constant to prevent typos (Esto huele a junior)